robot_id: debug
arms:
  arm_1:
    gripper_link: arm_1_link
    base_link: arm_1_base_link
    min_range: 0.2
    max_range: 0.4
    capabilities:
      pick_place: yes
      drill: yes
      interactive_mode: yes
      manipulation: yes
      move_to_user: yes
      get_ready: yes
  arm_2:
    gripper_link: arm_2_link
    base_link: arm_2_base_link
    min_range: 0.2
    max_range: 0.4
    capabilities:
      pick_place: yes
      drill: yes
      interactive_mode: yes
      manipulation: yes
      move_to_user: yes
      get_ready: yes